web/src/components/datasets/DatasetDetailPage.tsx

```diff
@@ -147,9 +170,31 @@ const DatasetDetailPage: FunctionComponent<IProps> = props => {
               />
             </Tabs>
           </Box>
-          <IconButton onClick={() => history.push('/datasets')}>
-            <CloseIcon />
-          </IconButton>
+          <Box display={'flex'} alignItems={'center'}>
+            <Box mr={1}>
+              <Button
+                color='primary'
+                className={classes.buttonDelete}
+                onClick={() => {
+                  props.dialogToggle('')
+                }}
+              >
+                {i18next.t('jobs.delete')}
+              </Button>
+              <Dialog
```

Comment:
There are two instances of this dialog... Can the dialog itself just be a reusable component.

Reply:
This component is reusable and located in `"components/Dialog.tsx"`, and use it with different props